### DISCUSSION TITLE
Render outline lines with 4 spaces indentation

### DISCUSSION THREAD

**Discussion by danieltomasz** [vslinko/obsidian-outliner#57] [Archive]
**Describe the bug**
Copied text of outline with 4 spaces  indicating subitem doesn't render lines

**To Reproduce**

Copy this text

```
-   problem-solving techniques:
    -   **Always have a plan**.
```
It will render and behave as list but the lines are not rendered

**Expected behaviour**
The lines will be rendered
![image](https://user-images.githubusercontent.com/7980381/113412339-4244c200-93b8-11eb-91e9-019ca53ee141.png)

**Screenshots**

![image](https://user-images.githubusercontent.com/7980381/113412162-dbbfa400-93b7-11eb-9026-054d0e72c7a2.png)

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Linux Os
 - Obsidian Version: [e.g. 22] Obsidian 0.11.9
 - Plugin Version 1.0.23

**Additional context**
Make an outline directly in the Obsidian work fine.
If this duplicate other issue or is expected due to some limitation it may be closed;

**Comment by vslinko** [comment]
@danieltomasz Thank you for your contribution!
It doesn't have a simple fix because I relate on `<span class="cm-tab"></span>` that codemirror creates for tabs but not for spaces.
I want to fix it but in some future.

Changing issue type from `bug` to `enhancement`

**Comment by vslinko** [comment]
I've converted the issue into discussions because it's very deep in my backlog and because it's possible to vote in discussions.

**Comment by vslinko** [comment]
Should be fixed in https://github.com/vslinko/obsidian-outliner/releases/tag/2.1.0
